packages/react-table/src/demos/Table.md

```diff
@@ -27,56 +16,6 @@ import globalDangerColor200 from '@patternfly/react-tokens/dist/js/global_danger
 import imgBrand from '@patternfly/react-core/src/components/Brand/examples/pfLogo.svg';
 import imgAvatar from '@patternfly/react-core/src/components/Avatar/examples/avatarImg.svg';
 
-import bulkSelectImg from './img/bulk-selector.png';
-import bulkSelectCheckboxImg from './img/bulk-select-checkbox.png';
-import bulkSelectExampleImg from './img/bulk-selection-example.png';
-import bulkSelectNoToolbarImg from './img/bulk-selection-no-toolbar.png';
-
-## Bulk selection
-Use a bulk selection pattern when you want to select or deselect multiple items in a content view (list, table, or card grid). The bulk selector uses a [Split button](/documentation/react/components/dropdown#split-button) component to control selection from the [Toolbar](/documentation/react/components/toolbar). Besides controlling selection, the bulk selector reflects the selection status of the related component (partially selected, all items selected, or no items selected).
```

Comment:
shouldn't this documentation be preserved?

Reply:
Yes, it's been moved to https://www.patternfly.org/v4/guidelines/bulk-selection per @mcarrano 's request.